src/data_storage/import/container_import.ts

```diff
@@ -1,24 +1,31 @@
 import axios, { AxiosRequestConfig } from "axios";
 import ContainerStorage from "../../data_storage/container_storage";
-import {CreateContainer} from "../../api_handlers/container";
+import { CreateContainer } from "../../api_handlers/container";
 import MetatypeRelationshipStorage from "../../data_storage/metatype_relationship_storage"
 import MetatypeStorage from "../../data_storage/metatype_storage"
 import MetatypeRelationshipPairStorage from "../../data_storage/metatype_relationship_pair_storage"
 import MetatypeKeyStorage from "../../data_storage/metatype_key_storage"
+import MetatypeRelationshipKeyStorage from "../../data_storage/metatype_relationship_key_storage"
 import { UserT } from "../../types/user_management/userT";
 import Result from "../../result";
 import { ContainerImportT } from "../../types/import/containerImportT"
 import { MetatypeT } from "../../types/metatypeT";
 import { MetatypeRelationshipPairT } from "../../types/metatype_relationship_pairT";
-import { MetatypeKeyT } from "../../types/metatype_keyT";
+import { MetatypeKeyT, MetatypeKeysT } from "../../types/metatype_keyT";
 import { MetatypeRelationshipT } from "../../types/metatype_relationshipT";
+import NodeStorage from "../../data_storage/graph/node_storage"
+import EdgeStorage from "../../data_storage/graph/edge_storage"
+import Logger from "../../logger"
 const convert = require('xml-js');
```

Comment:
so can this accept an xml file as well as a .owl file?

Reply:
It could, although there will certainly be an issue with parsing the ontology from it. Do you think we should check for the .owl extension and error out early if it's anything else?